.github/workflows/build-release.yml

```diff
@@ -67,30 +67,20 @@ jobs:
       channel: ${{ inputs.channel }}
     secrets: inherit
 
-  publish-test-packages:
-    name: Publish Release Test Packages
-    needs:
-      - build-packages
-    uses: ./.github/workflows/publish-release.yml
-    with:
-      channel: "nightly"
-      run_id: ${{ github.run_id }}
-    secrets: inherit
-
```

Comment:
This was a bit of a hack, we published first to nightly and use the packages to build the examples. Even for release branch packages.